racun-service/src/main/java/rs/edu/raf/banka/racun/service/impl/SredstvaKapitalService.java

```diff
@@ -284,4 +286,28 @@ public AgentSredstvaKapitalDto findSredstvaKapitalAgent(String token) {
         }
            return null;
     }
+
+    public SredstvaKapital createMarzniRacun(UUID uuidRacuna, String kodValute, Long hartijaId){
```

Comment:
Ovu metodu nazovi pocetnoStanjeMarzniRacun ili initMarzniRacun. KodValute nam ne treba jer ce to biti dolar, ali nam treba KapitalType, kasnije cemo dodati initialMargin, loanValue i to ako bude potrebno, ali za sada od parametara neka budu uuidRacuna, kapitalType, hartijaId

Reply:
Umesto ovoga: 
Valuta valuta = valutaRepository.findValutaByKodValute(kodValute);
stavi: Valuta valuta = valutaRepository.findValutaByKodValute("USD");